### PR TITLE
Update node-syncing.md

### PR DESCRIPTION
**Pull request by ChuckWhitson**
Added additional GitHub repo / file link that has a constantly updated and verified list of nodes

### CODE REVIEW

File: docs/reference-client/node-syncing.md [hunk 1]
```diff
@@ -43,8 +43,10 @@ If you would like to speed up connecting to other nodes and syncing, add one of
 - Eastern North America `introducer-us-east-1.chia.net:8444`
 - Europe: `introducer-eu-west-2.chia.net:8444`
 
-There is a public node share the available 8444 peers every hour.
+Here is a GitHub repo / file that is frequently updated with verified nodes:
+- [GitHub.com/ChuckWhitson/Chia-Known-Nodes](https://raw.githubusercontent.com/ChuckWhitson/Chia-Known-Nodes/master/Currently_Available_Node_IPs.txt)
```
Comment (BrandtH22) [inline docs/reference-client/node-syncing.md:47]: Hey, thank you for the updates! Can you update this to a permanent link?

**Comment by danieljperry** [comment]
Thanks!

**Review by danieljperry — DISMISSED** [review]

**Review by BrandtH22 — APPROVED** [review]
lgtm, thank you very much @ChuckWhitson for working through all of those changes in the midst of our reorg, much appreciated!

**Comment by BrandtH22** [comment]
@ChuckWhitson , I was about to perform the merge but noticed that the last commit was not signed [d910bb5](https://github.com/Chia-Network/chia-docs/pull/502/commits/d910bb5492fe7a8a9d7a6618bde5a3ae634a6968) .

If you can either revert that commit or resubmit signed I will get this pr merged, thank you!

**Comment by BrandtH22** [comment]
> > @ChuckWhitson , I was about to perform the merge but noticed that the last commit was not signed [d910bb5](https://github.com/Chia-Network/chia-docs/pull/502/commits/d910bb5492fe7a8a9d7a6618bde5a3ae634a6968) .
> > If you can either revert that commit or resubmit signed I will get this pr merged, thank you!
> 
> @BrandtH22, my apologies :) - new code box and didn't have my GitHub signing key configured.
> 
> I plan to take a more thorough review of the existing documentation so be on the lookout for more commits in the coming days. I'm curious - is the reorg in a complete / final state? If not, I can hold off on future commits until the reorg is finalized to limit the confusion.

Thank you very much! Once it makes it past the workflow I will get this merged 👍 

Yes the docs reorg is in its final state for the docs site (still working on some stuff for the chialisp site) so please do make prs for content or additional info. 

Insights into what I am doing on the docs site:

You will note I have been adding things to the issues list here on gh which are currently on my backlog so there is some low hanging fruit there. 

My main focus over the next two weeks or so is to get prettier (the auto-formatting solution) and crowdin (the community based translation solution) updated and working properly. 

After which I am planning to create a new section for contributions that would be at the head of the docs hierarchy

**Review by BrandtH22 — APPROVED** [review]
lgtm

**Comment by BrandtH22** [comment]
Hey @ChuckWhitson , it looks like those other commits are still not signed. 
Essentially we either need to revert those previous unsigned commits or resubmit the pr with all commits signed, once done then GH will be happy and let me merge. 

Let me know if any questions arise :+1:

**Pull request closed without merging**